### PR TITLE
feat: [SSCA-1977]: Add icon for rule definition

### PR DESCRIPTION
**Pull request by deepesh-ui**
You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2508

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2509

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2510

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2511

**Review by reetika04 — APPROVED** [review]

**Review by karansaraswat19 — APPROVED** [review]

File: packages/icons/src/cis.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...SIAESKALE6CAlePinvXAq8fYMZlYaptJ6Ar8AmywrGQXYQ4ByxXCIKCp0BWFzdbGoG39FflXB4yzxnacdcFDardMHkYNVw70TCIBEiABEiCBA5oABaw8Lt+FC+GwNFwyJmRZ5wTFTIDgdESeAhZstSyBgOUKXbblxLC5FVqut9Ct+IYx1tqgFX/TEusdu3u3D84c2mcXvMpjDh8GEiABEiABEiCBA5kABawCr973H3pjUNDExoeM9TXITSNgg9VPbbFSughTNViZApZ6hUDehGsIdCGqsXwM9lsfoytyi22bjUjbADv7d+yA2QhRa2vQ2NtDJU6NE7X3HCI1dYMHD65vjxDmOA5MzKQES3cscdTF7ssC7wFmJwESIAESIIHWCFDAao1QjvSrH133mW6B2Fgj5nSMLDwJctNAaKcKFbBgJJ/wt6XCl23b8IOqtlyw6TImCiFsF4S3XRDEdkEA+wRZdmAfey3b7AnY1i7sey/Kq/uIRlQTc4U3W2z8IllC8PVViq7NHhCkuluWVYrT6YmlD5a+WJZiuRhpnOgaIBhIgARIgARIoKMIUMDqIJI3Pv7GoLBtDYdQMw4CzQgIPUMhGJVldBF6abByCVgwnIfwBRUZhC13XYUvdFW6BvU2dmYbrCfTNG9CMJNgAHow5NX8ql3T44AglXm2LyNiDOL3ZCZwmwRIgARIgARIoO0Est64ba+KJZME5jzxz54NofhnQrY9HALSiJBtHQ+Ba1DAcqBFckcaqmsHV2CCZmp/CljrccyjIWB9kDx2/pIACZAACZAACbSfAAWs9jNstYZfV73buyTgHGU58eMCAfOFoLGOg1x1NISv/hCwuuXoIuxsDdZ2HLwKWPRI3+pVZAYSIAESIAESyJ8ABaz8WXVYzkWLHDt65Ia+4XDgSGi0hgYdaygMpoZCq3Uk7KwGQPDqDaP3Xuj6C2Pd5OoihOcIVwOGfAlhTH/RFZhnF6HaXp0CAau6w06OFZEACZAACZAACcAEmqFoCFRu2FAS292tV4kV72ts018Cdj84LT1cRypCehpgG6ccwlBvbPeEQNUdNlhw0QUhzFgwyzKwa4cwhi5IbDsQ2OCDS9TwvR5aslok7kHZnTjZGizbsGi3oC73IX4zfhlIgARIgARIgAQ6iMD/A3QIrdfFS1KyAAAAAElFTkSuQmCC"/></defs></svg>
```
Comment (tkAcharya007) [inline packages/icons/src/cis.svg:1]: Should not this be in same format as ssca-ruledefinition.svg?

**Pull request merged**